go.mod

```diff
@@ -1,6 +1,6 @@
 module oras.land/oras
 
-go 1.24.0
+go 1.24.2
```

Comment:
nit: it seems not necessary for this upgrade.

Reply:
We usually do it before each release.